zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanData.java

```diff
@@ -148,8 +161,8 @@ public TechSet getTechSet() {
      * @param cpType the custom page type to be checked
      * @return {@code true} if the message matches, {@code false} otherwise
      */
-    private boolean isCustomPage(HttpMessage msg, CustomPage.Type cpType) {
-        if (context == null) {
+    boolean isCustomPage(HttpMessage msg, CustomPage.Type cpType) {
```

Comment:
Changes on these two lines are probably unnecessary in the end, will revert shortly.